From: uazo <uazo@users.noreply.github.com>
Date: Wed, 22 May 2024 07:43:04 +0000
Subject: Keep disabled WebViewMediaIntegrityApi

Disables a new mechanism similar to the Web Environment Integrity API (https://wikipedia.org/wiki/Web_Environment_Integrity) used in WebViews (See https://issuetracker.google.com/issues/327186031).

More info: https://android-developers.googleblog.com/2023/11/increasing-trust-for-embedded-media.html.

Initial Commit: https://github.com/uazo/cromite/commit/feccf71b014a8dfcf04fbd8bf826ea000d5d3b7d
Category: Privacy
License: GPL-2.0-or-later - https://spdx.org/licenses/GPL-2.0-or-later.html
Requires: Add-cromite-flags-support.patch
---
 android_webview/common/aw_features.cc | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/android_webview/common/aw_features.cc b/android_webview/common/aw_features.cc
--- a/android_webview/common/aw_features.cc
+++ b/android_webview/common/aw_features.cc
@@ -258,5 +258,7 @@ BASE_FEATURE(kCreateSpareRendererOnBrowserContextCreation,
              "CreateSpareRendererOnBrowserContextCreation",
              base::FEATURE_DISABLED_BY_DEFAULT);
 
+SET_CROMITE_FEATURE_DISABLED(kWebViewMediaIntegrityApi);
+SET_CROMITE_FEATURE_DISABLED(kWebViewMediaIntegrityApiBlinkExtension);
 }  // namespace features
 }  // namespace android_webview
--
